src/theme/Footer/index.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Link from '@docusaurus/Link';
+import { useThemeConfig } from '@docusaurus/theme-common';
+
+const socialLinks = [
+  { bg: 'bg-git-hub', link: 'https://github.com/Drill4J' },
+  { bg: 'bg-telegram', link: 'https://t.me/drill4j' },
+  { bg: 'bg-twiter', link: 'https://twitter.com/' },
+  { bg: 'bg-youtube', link: 'https://www.youtube.com/channel/UCJtegUnUHr0bO6icF1CYjKw/featured' },
+];
+
+const Footer = () => {
+  const { footer: { links } } = useThemeConfig();
+  return (
+    <footer className="bg-monochrome-light-tint">
+      <div className="container">
+        <section className="grid py-14 grid-cols-1 md:grid-cols-2 lg:grid-cols-footer" style={{ gap: '106px' }}>
```

Comment:
Are you sure that there should be `106px`? And maybe update tailwindConfig for it?